samples/eth_inrapA.json

```diff
@@ -0,0 +1,9 @@
+{
+  "_id": "6b56ee657c870dfacd34e9ae4e050fcc",
+  "dc_issued": "2012",
+  "dc_language": "fra",
+  "dc_title": "Restaurer la vapeur",
+  "dc_creator": "Christophe Lejeune",
+  "text": "Comment se coordonne la restauration de locomotives à vapeur ? Comment les bénévoles acquièrent-ils collectivement les compétences techniques et les habiletés sociales permettant de faire rouler chaque semaine des locomotives à vapeur ? Comment se construit l'identité de celui ou celle qui est prêt-e à passer ses week-ends à poncer des essieux de locomotives ? Comment cet investissement se maintient-il dans la durée ?",
+  "couchapp": {}

```

Comment:
This is an artefact from pushing the document with the command.  This shouldn't be in the original sample.